src/FileMigrationHelper.php

```diff
@@ -57,26 +73,36 @@ public function run($base = null)
         Environment::increaseMemoryLimitTo();
 
         // Loop over all files
-        $count = 0;
+        $processedCount = $migratedCount = 0;
         $originalState = null;
         if (class_exists(Versioned::class)) {
             $originalState = Versioned::get_reading_mode();
             Versioned::set_stage(Versioned::DRAFT);
         }
 
-        foreach ($this->getFileQuery() as $file) {
+        $query = $this->getFileQuery();
+        $total = $query->count();
+        foreach ($query as $file) {
             // Bypass the accessor and the filename from the column
             $filename = $file->getField('Filename');
 
             $success = $this->migrateFile($base, $file, $filename);
+
+            if ($processedCount % 100 === 0) {
```

Comment:
Are we concerned about database record size on a queuedjobs descriptor here? I think it should have a verbose mode where it logs every file move. It'll be an essential asset for later in case you have to diagnose what went wrong during a migration.

Reply:
It already tells you why a file was not migrated if that happens.